Week1/homework/homework.txt

```diff
@@ -0,0 +1,81 @@
+Here is the lucidChart diagram:
+https://www.lucidchart.com/documents/edit/00de9f5d-4578-420f-8fed-93e057398424/0
+
+Here are the attempts at the database queries.
+
+1. What are the names of all the cities in the Netherlands?
+
+	SELECT city.Name 
+	FROM city 
+	JOIN country 
+	ON city.CountryCode = country.Code 
+	WHERE country.Name = 'Netherlands';
+
+2. What is the population of Rotterdam ?
+
+	SELECT population 
+	FROM city 
+	WHERE name='Rotterdam';
+
+3. What's the name of all the countries on the continent ‘North America’ ?
+
+	SELECT name 
+	FROM country 
+	WHERE continent='North America';
+
+4. What's the top 10 countries by SurfaceArea ?
+
+	SELECT name 
+	FROM country 
+	ORDER BY SurfaceArea DESC
+	[alternatively, "GROUP BY SurfaceArea DESC"]
```

Comment:
Interesting point! It's better to stick to `ORDER BY` like you did though, because the use of `group by` in that way is quite unconventional, so it might be hard for other developers to understand what you are doing and maintain your code. The performance of it is also worse. 
Fun fact we didn't talk about in class: you can add comments to SQL by putting `--` before the comment